tools/sdk-ai-bots/online-evaluation.yml

```diff
@@ -8,7 +8,7 @@ schedules:
     displayName: "Weekly Monday Morning Trigger"
     branches:
       include:
-        - azure-sdk-ai-bot
+        - main
```

Comment:
Could you also add this branch which is for our hotfix:  https://github.com/Azure/azure-sdk-tools/tree/release/qa-bot